pdata/testdata/profile.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package testdata // import "go.opentelemetry.io/collector/pdata/testdata"
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pprofile"
+)
+
+var (
+	profileStartTimestamp = pcommon.NewTimestampFromTime(time.Date(2020, 2, 11, 20, 26, 12, 321, time.UTC))
+	profileEndTimestamp   = pcommon.NewTimestampFromTime(time.Date(2020, 2, 11, 20, 26, 13, 789, time.UTC))
+)
+
+// GenerateProfiles generates dummy profiling data for tests
+//
+// # Experimental
+//
+// Notice: This method is EXPERIMENTAL and may be changed or removed in a
+// later release.
```

Comment:
`pdata/testdata` is not stable https://github.com/open-telemetry/opentelemetry-collector/blob/05a2844bdb4f4483efed9eca9c3d6e54c859ebaa/versions.yaml#L50 so IMO we don't need this. We are also (@open-telemetry/collector-approvers to correct me here if I am wrong) targeting this one for 1.0, so I think adding the function here is fine.